multiversx_sdk_cli/config.py

```diff
@@ -202,24 +202,11 @@ def add_config_args(argv: List[str]) -> List[str]:
     except KeyError:
         return argv
 
-    check_for_deprecated_args(config_args)
-
     final_args = determine_final_args(argv, config_args)
     print(f"Found extra arguments in mxpy.json. Final arguments: {final_args}")
     return final_args
 
 
-def check_for_deprecated_args(args: List[str]) -> None:
```

Comment:
Actually, before an actual removal, let's convert them to errors - `throw new KnownError ...`. People should not be using them anymore.

Reply:
I've deleted this because we have another function called `verify_deprecated_entries_in_config_file()` that was performing the same check.

Reply:
Great!